lib/audio-device.js

```diff
@@ -64,6 +64,18 @@ class SDLAudioDevice {
     })
   }
 
+  static getRecordingDevice({ name } = {}) {
+    if (!name) {
+      return SDLAudioDevice.defaultRecordingDevice()
+    }
+
+    return new SDLAudioDevice(
+      SDLAudioDevice.recordingDevices().find((device) => {
+        return name === device.name
+      })
+    )
```

Comment:
What if we don't find a device there? Maybe could we use `return SDLAudioDevice.defaultRecordingDevice()` too?